languages/java/src/main/java/net/minecraftforge/fml/javafmlmod/FMLModContainer.java

```diff
@@ -70,6 +71,7 @@ private void constructMod()
         }
         catch (Throwable e)
         {
+            if (e instanceof InvocationTargetException) e = e.getCause();
```

Comment:
might be nice to add a comment above to explain that this is done for the error screen